rust/src/lib.rs

```diff
@@ -381,74 +371,72 @@ pub unsafe extern "C" fn zcashlc_derive_extended_spending_keys(
 /// - `encoding` must be non-null and must point to a null-terminated UTF-8 string.
 ///   and it must be aligned to the size of a u32.
```

Comment:
https://github.com/zcash-hackworks/zcash-light-client-ffi/pull/35/files#r955098800

Reply:
In any case, whenever you allocate a `repr(C)` struct, the assignment will be correct according to the C ABI. That's not a safety constraint that API users are responsible for ensuring.